Gruntfile.js

```diff
@@ -444,7 +455,7 @@ module.exports = function (grunt) {
             `electron-builder.template.yaml`,
         );
 
-        const version = grunt.option('unified-version') || '0.0.0';
+        const version = getUnifiedVersion();
```

Comment:
it feels odd to me that we check for existence above in line 392, but not here - I'd be fine with either as long as it's consistent

Reply:
You're correct. I had moved this into getUnifiedVersion, then pulled it out and forgot to restore this one.